kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/ClusterCondition.java

```diff
@@ -26,6 +26,11 @@ static ClusterCondition filterNotExists(String cluster, String filterName) {
                 String.format("Filter \"%s\" does not exist.", filterName));
     }
 
+    static ClusterCondition ingressNotAvailable(String cluster, String ingressName) {
+        return new ClusterCondition(cluster, ConditionType.Accepted, Status.FALSE, INVALID,
+                String.format("Ingress \"%s\" is non-existent.", ingressName));
```

Comment:
```suggestion
                String.format("KafkaProxyIngress \"%s\" is non-existent.", ingressName));
```

Reply:
going to delete this method and re-introduce in a subsequent PR adding the cluster condition